### DISCUSSION TITLE
Why can't I send emails to outlook with postal? I need assistance.

### DISCUSSION THREAD

**Discussion by ghostverse** [postalserver/postal#3024] [Help with using Postal]
I made the necessary DNS configurations to send emails with Postal and tested with Mail-Tester.com and MX Toolbox for deliverability assessment. Everything seems fine, but when I send an email to Outlook, I got blacklisted. The domain and IP are clean and have no history of spam; just one email sent to Outlook got me blacklisted. How can I fix this?

**Comment by willpower232** [comment]
Unfortunately they are a law unto themselves, you can ask the recipient to mark the message as not spam which may help.

You can also check the basics like having a PTR record (reverse DNS) set up and making sure that port 25 is open to everyone.

**Reply by ghostverse** [reply]
The Outlook email address is mine, and I did not receive an email from the SMTP server. The PTR record is set properly, and port 25 is open on the server. Initially, I had issues with the PTR record as it was using the server's IPv6 address. I then disabled IPv6 to use an IPv4 address, which helped deliver to Gmail's inbox. However, Outlook has been problematic, adding me to their blocklist for no apparent reason. 
At this point, what do you suggest I do?

**Reply by willpower232** [reply]
Unless you're able to open a support ticket to them, there isn't much you can do. It may be they have rules against certain IP ranges or relatively new domain names.

**Reply by ghostverse** [reply]
The domain tends to be older.
